csrc/device_lower/pass/predicate.cpp

```diff
@@ -67,7 +67,11 @@ class ConditionalFromPredicateModifier : public kir::ExprMutator {
           auto vec_expr = ite->thenBody()[0];
           NVF_ERROR(
               vec_expr->isA<UnaryOp>() || vec_expr->isA<LoadStoreOp>() ||
-                  vec_expr->isA<TernaryOp>() || vec_expr->isA<IndexSelectOp>(),
+                  vec_expr->isA<TernaryOp>() ||
+                  vec_expr->isA<IndexSelectOp>() ||
+                  // To supress the throw.
+                  // I think this is predicated on the vectorized dim.
+                  vec_expr->isA<BlockQuantizationOp>(),
```

Comment:
Not sure why. Can you give more details?

Reply:
I am looking into this.

Reply:
I think this is okay. 
We do want this Op to work on 4/8 consecutive elements per thread. So it should be okay to have it vectorized. Please let me know if this is wrong.

Reply:
Are we still using Vectorize with BlockQuantizationOp?